application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -45,24 +44,25 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
+        int MAX_TAGS_THRESHOLD_WARNING = 200;
+        if (tagSystem.getAllIds().size() > MAX_TAGS_THRESHOLD_WARNING) {
+            // TODO Implement the edge case
 
-        ArrayList<String> list = new ArrayList<>(tagSystem.getAllIds());
-
-        if (list.size() > 200) {
-
-            Logger logger = Logger.getLogger(TagsCommand.class.getName());
-
-            logger.setLevel(Level.WARNING);
-            logger.warning("- WARNING - TAGS ARE BEYOND 200 LINES ");
+            logger.warn(
+                    "The amount of tags is very high and it might soon exceed the maximum character limit. The code should be adjusted to support this edge case soon.\n");
```

Comment:
Dont add a `\n`. It ensures that stuff on its own.

Reply:
Must have slipped when I copied the feedback message